src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -184,6 +188,32 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     } yield callToLogsMap
   }
 
+  private def workflowMetadata(id: WorkflowId): Future[WorkflowMetadataResponse] = {
+    for {
+      callStandardStreamsMap <- workflowStdoutStderr(id)
```

Comment:
Not a complaint but curious why this is called `Streams`. I thought it had to do with the data type but that doesn't seem to be the case